eth_tester/validation/outbound.py

```diff
@@ -277,6 +312,19 @@ def _validate_fork_specific_fields(block):
         partial(validate_array, validator=validate_withdrawal)(block["withdrawals"])
         validate_32_byte_string(block["withdrawals_root"])
 
+    # Cancun fork
+    if all(
+        _ not in block
```

Comment:
I like @reedsa's suggestion to make these into methods. I think it would be easier to reason about.